usr/share/rear/restore/NFS_SERVER/default/400_restore_with_nfs_server.sh

```diff
@@ -0,0 +1,19 @@
+# 400_restore_with_nfs_server.sh
+
+local check_file=$TARGET_FS_ROOT/$NFS_SERVER_RESTORE_FINISHED_FILE
+
+Print "Mount the nfs share: 'mount -t nfs -o nfsvers=4 <ip>:/ <destination>' and restore all files to mounted destination."
+Print "Create the $check_file file when the restore is completed and umount the share."
+
+rm -rf $check_file
```

Comment:
```suggestion
rm -f $check_file || Error "Couldn't delete restore finished file"
```

If this happens to be a directory then we 100% for sure have a conflict and should abort.

Reply:
Yes, please no `rm -r` unless it is actually needed.
FYI
what (not so) fun we had with careless `rm -rf` cf.
https://github.com/rear/rear/issues/2611